projects/aca-shared/rules/src/navigation.rules.ts

```diff
@@ -110,6 +110,11 @@ export function isLibraryContent(context: RuleContext): boolean {
   return url && (url.endsWith('/libraries') || url.includes('/libraries/') || url.startsWith('/search-libraries'));
 }
 
+export function isDetails(context: RuleContext): boolean {
+  const { url } = context.navigation;
+  return url && (url.endsWith('/details') || url.includes('/details/') || url.startsWith('/details'));
```

Comment:
Can it be shorter, something like? 
return url?.includes('/details');

Includes is able to check if string is at the beginning or at the end too

Reply:
I have updated